examples/src/main/python/ml/aft_survival_regression.py

```diff
@@ -17,9 +17,9 @@
 
 from __future__ import print_function
 
+from pyspark.ml.linalg import Vectors
 # $example on$
 from pyspark.ml.regression import AFTSurvivalRegression
-from pyspark.ml.linalg import Vectors
```

Comment:
In that case, move the `# $example on$` comment up above the `from pyspark.ml.linalg import Vectors`